test/token/ERC721/extensions/ERC721Consecutive.t.sol

```diff
@@ -31,18 +34,34 @@ contract ERC721ConsecutiveTarget is StdUtils, ERC721Consecutive {
 }
 
 contract ERC721ConsecutiveTest is Test {
+    function test_starting_id(address receiver, uint256[] calldata batches, uint96 startingTokenId) public {
+        vm.assume(receiver != address(0));
+        vm.assume(startingTokenId > 0);
+        vm.assume(startingTokenId < 5000);
```

Comment:
Can we use [bound](https://book.getfoundry.sh/reference/forge-std/bound?highlight=bound#bound) instead?